gsa/src/gmp/models/tag.js

```diff
@@ -34,13 +33,10 @@ class Tag extends Model {
   parseProperties(elem) {
     const ret = super.parseProperties(elem);
     if (isDefined(elem.resources)) {
-      ret.resources = map(ret.resources.resource,
-        res => new Model(res, elem.resources.type));
       ret.resource_type = elem.resources.type;
       ret.resource_count = elem.resources.count.total;
```

Comment:
Would be nice to change resource_type and resource_count to camelCase and change resourceType to return a GSA type (by using `normalizeType` here) in a next step.